package.json

```diff
@@ -0,0 +1,755 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "private": "true",
+  "scripts": {
+    "test": "jest --coverage",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^29.5.0",
+    "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^2.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```

Comment:
- Please ensure `Jest` is present in your `devDependencies` so your code can be tested 👍🏽
![Screenshot from 2023-03-15 19-29-52](https://user-images.githubusercontent.com/44624138/225408710-f245830f-71b8-4b39-bd1c-d8afcebcdd98.png)
- You can add it by pasting the below dependencies into your devDependencies
```
"jest": "^29.4.3",
"jest-localstorage-mock": "^2.4.26",
```

Reply:
Addition of jest in [`dependencies`](https://github.com/Kel-nana/Review-To-do-list/blob/8f0fded52eaa5b5f4d5616b95be5dec0809d6807/package.json#L29-L31)